alfresco-search/src/test/java/org/alfresco/solr/highlight/AlfrescoHighlighterTest.java

```diff
@@ -310,23 +310,6 @@ public void testHighlighting() throws Exception
 
 
         logger.info("######### MultiTerm ###########");
-
```

Comment:
I believe it would be better to synthesise the reason of this change with a comment